bundle/src/deepracer_simulation_environment/scripts/mp4_saving/training_image_editing.py

```diff
@@ -58,6 +62,69 @@ def _edit_major_cv_image(self, major_cv_image, cur_training_phase):
         # Applying gradient to whole major image and then writing text
         major_cv_image = utils.apply_gradient(major_cv_image, self.gradient_alpha_rgb_mul,
                                               self.one_minus_gradient_alpha)
+        
+        if rospy.get_param('ENABLE_EXTRA_KVS_OVERLAY', 'False').lower() in ('true'):
+            width, height = IconographicImageSize.FULL_IMAGE_SIZE.value
+            major_cv_image = utils.plot_rectangle(major_cv_image, 0, 0, width, 85, RaceCarColorToRGB.Black.value )
+            
+            # Top left location of the picture
+            loc_x, loc_y = XYPixelLoc.SINGLE_AGENT_DISPLAY_NAME_LOC.value
+            
+            # Best lap time
+            best_lap_time = mp4_video_metrics_info[self.racecar_index].best_lap_time
+            # The initial default best_lap_time from s3_metrics.py is inf
+            # If the ros service in s3_metrics.py has not come up yet, best_lap_time is 0
+            best_lap_time = utils.milliseconds_to_timeformat(
+                datetime.timedelta(milliseconds=best_lap_time)) \
+                if best_lap_time != float("inf") and best_lap_time != 0 else "--:--.---"    
+            best_lap_time_text = "Best lap | {}".format(best_lap_time) 
+            major_cv_image = utils.write_text_on_image(image=major_cv_image, text=best_lap_time_text,
+                                                    loc=(width-180, loc_y), font=self.amazon_ember_light_18px,
+                                                    font_color=RaceCarColorToRGB.White.value,
+                                                    font_shadow_color=RaceCarColorToRGB.Black.value)
+            
+            # Last lap time
+            last_lap_time = mp4_video_metrics_info[self.racecar_index].last_lap_time
+            # The initial default last_lap_time from s3_metrics.py is inf
+            # If the ros service in s3_metrics.py has not come up yet, last_lap_time is 0
+            last_lap_time = utils.milliseconds_to_timeformat(
+                datetime.timedelta(milliseconds=last_lap_time)) \
+                if last_lap_time != float("inf") and last_lap_time != 0 else "--:--.---"    
+            last_lap_time_text = "Last lap | {}".format(last_lap_time) 
+            major_cv_image = utils.write_text_on_image(image=major_cv_image, text=last_lap_time_text,
+                                                    loc=(width-180, loc_y+25), font=self.amazon_ember_light_18px,
+                                                    font_color=RaceCarColorToRGB.White.value,
+                                                    font_shadow_color=RaceCarColorToRGB.Black.value)
+            
+            # Progress
+            progress_text = "Progress | {}%".format(round(mp4_video_metrics_info[self.racecar_index].completion_percentage,2)) 
```

Comment:
Cleaner is to do the rounding directly in format: 
`"Progress | {:.2f}%".format(mp4_video_metrics_info[self.racecar_index].completion_percentage)